decimal.go

```diff
@@ -107,10 +117,18 @@ func (c *Context) NewFromString(s string) (*Decimal, Condition, error) {
 }
 
 // String is a wrapper of ToSci.
+// REVIEW: why?
```

Comment:
ToSci in the first place because the GDA spec has both a ToSci and ToEng formatter (http://speleotrove.com/decimal/daconvs.html), and I wanted our API to reflect closely that spec. I haven't done ToEng yet, which is why ToSci alone seems weird.